src/main/java/ai/elimu/rest/v1/project/AppCollectionRestController.java

```diff
@@ -138,4 +145,61 @@ public String getApplications(
         logger.info("jsonObject: " + jsonObject);
         return jsonObject.toString();
     }
+    
+    /**
+     * As AppCollections in Custom Projects do not include all the Applications from the open source 
+     * version, some apps are required to form the basic infrastructure.
+     */
+    private void addInfrastructureApps(JSONArray applications) {
+        // Appstore
+        Application applicationAppstore = applicationDao.readByPackageName(Locale.EN, "ai.elimu.appstore");
```

Comment:
The logic is almost the same for 3 blocks: AppStore, Analytics and Custom Launcher, except for package name.
Is it better to move this logic to a function receiving package name as parameter instead?

Reply:
Good point. Will fix this.